extras/nginx/README.md

```diff
@@ -89,7 +89,7 @@ http {
         ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3;
 
         # Add Alt-Svc header to negotiate HTTP/3.
-        add_header alt-svc 'h3-25=":443"; ma=86400';
+        add_header alt-svc 'h3-27=":443"; ma=86400';
```

Comment:
Also, there is `NGX_HTTP_V3_ALPN_ADVERTISE` defined in nginx patch but seems not used anywhere.

```
nginx-1.16.patch:+#define NGX_HTTP_V3_ALPN_ADVERTISE       "\x05h3-18"
```